### PR TITLE
Initial setup

### PR DESCRIPTION
**Pull request by afmicc**
## Initial setup

* Add the project structure 
* Basic configurations

### CODE REVIEW

**Review by guilleijo — APPROVED** [review]

**Review by kaozdl — APPROVED** [review]

File: target_api/settings.py [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = '6=zw0+c5d7h4w^zfj$$z86wu=pcrsfa0z4#8!4kg%^#ef$+=0y'
```
Comment (nsantos16) [inline target_api/settings.py:23]: Maybe you can add an environment variable here

File: target_api/settings.py [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+ALLOWED_HOSTS = []
```
Comment (nsantos16) [inline target_api/settings.py:28]: Same comment with this, also in the future you can divide this setting file into `base.py`, `local.py` and `production.py`

File: target_api/settings.py [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+USE_TZ = True
+
+
+# Static files (CSS, JavaScript, Images)
+# https://docs.djangoproject.com/en/3.0/howto/static-files/
+
+STATIC_URL = '/static/'
+
+# django-rest-framework
+REST_FRAMEWORK = {
```
Comment (nsantos16) [inline target_api/settings.py:125]: Create a requirements.txt with all the requirements that you will add to the project. Also, you can use `Pipenv` to maintain your dependencies

File: target_api/settings.py [hunk 4]
```diff
@@ -0,0 +1,128 @@
...
+]
+
+WSGI_APPLICATION = 'target_api.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/3.0/ref/settings/#databases
+
+DATABASES = {
+    'default': {
```
Comment (nsantos16) [inline target_api/settings.py:78]: Same comment here, this will be an environment variable as well

**Review by nsantos16 — APPROVED** [review]
Great job, I left some comments

File: target_api/settings.py (hunk 1 above)
Comment (afmicc) [reply target_api/settings.py:23]: I do it in the next PR, please check #2

File: target_api/settings.py (hunk 2 above)
Comment (afmicc) [reply target_api/settings.py:28]: I do it something similar using `environ` in the next PR

File: target_api/settings.py (hunk 3 above)
Comment (afmicc) [reply target_api/settings.py:125]: same here

File: target_api/settings.py (hunk 4 above)
Comment (afmicc) [reply target_api/settings.py:78]: same here

**Pull request merged**